modules/users/client/components/BottomNavigationSmall.component.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * Bottom navigation for small screens
+ */
+
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { useTranslation } from 'react-i18next';
+import classNames from 'classnames';
+
+export default function BottomNavigationSmall({ username, isSelf, contactCount }) {
+  // @TODO the default value for 'active' will need to be fetched in more React way
+  // when we have a router in place
+  const [active, setActive] = useState(window.location.pathname.split('/')[3] || 'about');
+
+  const { t } = useTranslation('user');
+
+  const tabs = [
+    {
+      key: 'overview',
+      label: t('Overview'),
+      link: `/profile/${username}/overview`
+    },
+    {
+      key: 'about',
+      label: t('About'),
+      link: `/profile/${username}`
+    },
+    {
+      key: 'accommodation',
+      label: t('Hosting'),
+      link: `/profile/${username}/accommodation`
+    }
+  ];
+
+  // contacts tab is shown only sometimes
+  if (contactCount > 0 || isSelf) {
+    tabs.push({
+      key: 'contacts',
+      label: <>{t('Contacts')} <span className="badge">{contactCount}</span></>,
```

Comment:
This is fine to merge as is, just noting that another way would be to include `count: contactCount` and move the HTML to `return`

Reply:
Interesting. I didn't know that the span was ignored without content. I like this solution, as it separates label from count (which are two different things).